doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -440,11 +386,3 @@ Below is the list of system part properties.
 |Endpoint public key      | Public key used for security purposes to validate endpoint requests.              |
 |Endpoint group state     | List of endpoint groups that contain current endpoint.                            |
 |Client-side profile hash | Client-side profile hash used to validate integrity of endpoint profile state.    |
-
-## Further reading
```

Comment:
Generally, further reading table should be there. It just has to be relevant. Not important for 0.10 release.